frontend/packages/data-portal/app/components/common/CustomBanner.tsx

```diff
@@ -0,0 +1,62 @@
+import { Banner, Icon } from '@czi-sds/components'
```

Comment:
This looks really similar to `SurveyBanner.tsx`. Do you think we could instead reuse that component with a slight change. Something like there is a base banner called something like `TimeBasedBanner`.
And then SurveyBanner and NeuroglancerBanner (not CustomBanner) use this with some small set of props? Similar to the idea in `reusableBanner`